src/app/Attributes/Button.php

```diff
@@ -4,11 +4,12 @@
 
 class Button
 {
-    public const Mandatory = ['class', 'icon', 'type'];
```

Comment:
why we don't want icon to be mandatory anymore?

Reply:
for slots, we don't need icon

Reply:
Yes, but for the rest we do. If we remove icon from mandatory the validation makes no sens.

Also, we don't mind adding icons for custom slots in the template, we keep things in one place.

Reply:
if we set icon mandatory and we show the icon for slots too, in some cases like now that we want to use vpopover then we will get a problem, because we need to put icon inside another element

Reply:
we would still render the icon manually, wherever wee need it as long as we make it available in the slot props.

having the icon in template will only make things a bit consistent.

Reply:
oh, you are right it's much nicer, I'll fix it  now